### DISCUSSION TITLE
Earthquake in Japan

### DISCUSSION THREAD

**Discussion by ktgw0316** [Aries85/LightZone#335] [Announcements]
I just wanted to let you know that I am safe.

Context: On this New Year's Day, a big earthquake and tsunami hit Japan.
I and my family are staying far from the epicenter, so we are OK.  I hope the damage will not be severe...

**Comment by jedd** [comment]
Very glad to hear! Sounds like - so far at least - an awful lot of damage, but not many casualties, thanks to a lot of preparedness.

**Comment by sfink16** [comment]
Great news and stay safe!

**Comment by doobes** [comment]
Glad to hear you and your family are well.

Thank you for letting us know!
